src/modules/vtol_att_control/standard.cpp

```diff
@@ -89,39 +79,16 @@ void
 Standard::parameters_update()
 {
 	float v;
-	int i;
 
 	/* duration of a forwards transition to fw mode */
-	param_get(_params_handles_standard.front_trans_dur, &v);
-	_params_standard.front_trans_dur = math::constrain(v, 0.0f, 20.0f);
```

Comment:
@dagar  What do you think about these things? If we want them we need to add them to the vtol_att_control_main.cpp which handles most params now. The questions is if this makes sense in general?

Reply:
The intent is good, but I don't like that your actual parameter value isn't used. In many cases these constraint aren't necessary because the param metadata prevents invalid user data. At a minimum I'd want to see the effective parameter value committed.

Reply:
I agree to remove the constrain as UI will make clear not to go beyond that